subprojects/version-info/version-info.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+task buildReceiptResource(type: Copy, dependsOn: rootProject.createBuildReceipt) {
+    into "$generatedResourcesDir/org/gradle"
+    from rootProject.createBuildReceipt.receiptFile
```

Comment:
For our CI infrastructure we still want to have a top level build receipt in `/build/build-receipt.properties`. We load this file when doing a promotion. We could have the generation in this file and then move it to `/build`. I was actually thinking to move the generation of the test build receipt to `gradle/buildReceipt.gradle` and then copy it for the test `version-info.jar` to here to have all build receipt tasks in one place.